programming/Encoders.md

```diff
@@ -0,0 +1,57 @@
+<b>Ever been on a website written in another language and had to utilize google translate to gain some 
```

Comment:
Since we're using markdown, can we use `**text here**` rather than `<b>text here</b>`